### PR TITLE
Core 1368 Fix Sidebar scrolling

### PR DESCRIPTION
**Pull request by sri2k1us**
- Large Screen 
  
image

- Tour Prompt
image

Note:
Scrollbars were not showing up cos of some markups / sytles with react-joyride lib. I tried putting the `ProductTour` component in different fragments in `AppBar.js` and none them fixed it. So I took a different approach to fix this. I moved the product tour prompt to be a `Dashboard` item and all users will get the tour prompt until it is dismissed. We discussed a while back in UX meeting about providing a tour to all users. So I think this will solve this requirement.

### CODE REVIEW

File: src/components/layout/DrawerItem.js [hunk 1]
```diff
@@ -47,17 +46,6 @@ const DrawerItem = (props) => {
...
-                    <img
-                        className={
-                            clsxBase
-                                ? clsx(clsxBase, classes.drawerIcon)
-                                : classes.drawerIcon
-                        }
-                        src={image}
-                        alt={title}
-                    />
-                )}
```
Comment (sri2k1us) [inline src/components/layout/DrawerItem.js:60]: Since we use react components for custom SVG images, this is no longer needed.

**Review by psarando — APPROVED** [review]
LGTM :shipit: 👍

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**